client/layout/logged-out-oauth.jsx

```diff
@@ -3,17 +3,15 @@
  */
 import React from 'react';
 
-module.exports = React.createClass( {
-	displayName: 'LayoutLoggedOutAuth',
-
-	render: function() {
-		return (
-			<div className="wp logged-out-auth">
-				<div id="content" className="wp-content">
-					<div id="primary" className="wp-primary wp-section">
-					</div>
-				</div>
+const LayoutLoggedOutAuth = () => (
+	<div className="wp logged-out-auth">
```

Comment:
Just noting that the file's name is `logged-out-oauth`, but the CSS class and component display name refer to `auth`. Not sure which is right.